src/page-objects/utils.ts

```diff
@@ -9,72 +9,22 @@ export function getElementCenter(rect: ElementRect): { x: number; y: number } {
   };
 }
 
-// This is the pixel value *before* applying the device pixel ratio
```

Comment:
We always test for one device generation. There is no point of device detection.